build.py

```diff
@@ -134,8 +143,17 @@ def targets():
 
     if not "BUILD_ONLY" in os.environ:
         check_call(
-            "circuitpython-build-bundles --output_directory dist --filename_prefix circuitpython-fonts --library_location libraries/ --library_depth 1",
-            shell=True,
+            [
+                "circuitpython-build-bundles",
+                "--output_directory",
+                "dist",
+                "--filename_prefix",
+                "circuitpython-fonts",
+                "--library_location",
+                "libraries/",
+                "--library_depth",
+                "1",
```

Comment:
@jepler this fails `VersionInfo.is_valid` in circup. I this needs to be `1.0.0`

Reply:
I'm rereleasing with a version of 20240506.1. let me know what circup makes of this if you have a chance to test.

Reply:
I also had to introduce a dummy commit, because when I repeatedly tagged the same commit, the "earlier" (? shorter?) tag was chosen. Now 20240507.0.0 seems to wfm

```
$ circup bundle-show
...
adafruit/circuitpython-fonts
    https://github.com/adafruit/circuitpython-fonts
    version = 20240507.0.0
```
but I don't have any device handy to check if the library version handling is workng.

Reply:
@jepler success! Thank you.
<img width="863" alt="image" src="https://github.com/adafruit/circuitpython-fonts/assets/1547881/9d8930c5-d296-49ab-bef6-223825f3647e">